### ISSUE TITLE
Valid accuracy and loss are nans

### ISSUE THREAD

**Issue by oumayb** [oumayb/craters-detection-tensorflow#1]
With the current architecture, validation accuracy and loss are nans during training (training loss and accuracy have a normal behavior)

**Comment by oumayb** [comment]
I was inputting the network less validation images that the batch size. Inputting more images than the batch size value doesn't output nans anymore.
